src/service/dynamic_report/dynamic_report.py

```diff
@@ -136,7 +136,7 @@ def get_year_report(
         try:
             company = dict()
             is_valid_company = company_id and company_id.strip()
-            headers = ["name"]
+            headers = ["name", "size_cohort", "margin_group"]
```

Comment:
This change requires removing the size_cohort and margin_group from the header after the filter step.
Instead of that just add these fields in the remove_fields function -> 
```
def remove_fields(self, company_data: dict, headers: list) -> None:
    company_data.pop("prior_actuals_revenue", None)
    header = ["id", "size_cohort", "margin_group"]
....
```